src/main/java/dev/ecattez/shahmat/board/Colored.java

```diff
@@ -0,0 +1,7 @@
+package dev.ecattez.shahmat.board;
+
+public interface Colored {
+
+    boolean isOfColor(PieceColor color);
```

Comment:
`sameColorAs` ?

Reply:
It could work with a `Colored` parameter but it is useless